[IT] Update `Guides/How_to_use_the_Offset_Wizard` (#9597)
ppy/osu-wiki: wiki/Guides/How_to_use_the_Offset_Wizard/it.md

> # Come usare l'Offset Wizard

Reviewer: ```suggestion
# Come usare la Taratura Guidata dell'Offset 
```

*Vedi anche: [Offset Wizard](/wiki/Client/Options/Offset_Wizard) e [Offset universale](/wiki/Offset/Universal_offset)*

Se gli [hitsound](/wiki/Beatmapping/Hitsound) non risultano sincronizzati con la musica in ogni beatmap che giochi, molto probabilmente dovrai cambiare il tuo [offset universale](/wiki/Offset/Universal_offset). Il modo più facile per farlo è utilizzando l'**Offset Wizard**.

Segui i seguenti step per regolare il tuo offset universale utilizzando l'Offset Wizard.

## Aprire l'Offset Wizard

1. Apri osu!.
2. Vai al menù `Opzioni`, o premi `Ctrl` + `O`.
3. Cerca "offset" nella barra di ricerca rapida, e premi sul pulsante `Taratura guidata dell'offset`.

> L'Offset Wizard verrà quindi aperto, riproducendo la musica precedentemente giocata in osu!, mentre delle barre verdi scorrono sullo schermo e un metronomo tiene il ritmo.

Reviewer: ```suggestion
La procedura di taratura guidata dell'offset inizierà, riproducendo un brano precedentemente riprodotto, mentre delle barre verdi verticali scorreranno sullo schermo orizzontalmente. Si potrà udire anche un metronomo, per tenere il ritmo.
```

> ## Calibrare l'offset universale

Reviewer: ```suggestion
## Tarare l'offset universale
```

1. Ascolta attentamente la musica e il metronomo contemporaneamente.
2. Inizia a regolare l'offset universale su o giù, usando le corrispondenti freccette, o la rotellina del mouse.
   - Se gli hitsound ti sembrano troppo in anticipo rispetto alla musica, dovrai *diminuire* il tuo offset universale.
   - Se gli hitsound ti sembrano troppo in ritardo rispetto alla musica, dovrai *aumentare* il tuo offset universale.
3. Modificalo fin quando il ritmo del metronomo e la musica non combaciano perfettamente, aiutandoti con le barre che scorrono sullo schermo.

> 4. Una volta calibrato l'offset universale con successo, premi il pulsante `Indietro` in basso a sinistra dello schermo per uscire dall'Offset Wizard.

Reviewer: ```suggestion
4. Una volta calibrato l'offset universale con successo, premi il pulsante `Indietro` in basso a sinistra dello schermo per uscire dalla procedura di Taratura Guidata dell'Offset.
```
